base_manifest_extension/hooks.py

```diff
@@ -1,21 +1,32 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Therp BV <http://therp.nl>
+# Copyright 2017 LasLabs Inc.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
 
 import inspect
 from openerp.sql_db import Cursor
 from openerp.modules import module
 from openerp.modules.graph import Graph
+
 original = module.load_information_from_description_file
+rdepends_to_process = {}
 
 
 def load_information_from_description_file(module, mod_path=None):
     result = original(module, mod_path=mod_path)
-    # add the keys you want to react on here
-    if result.get('depends_if_installed'):
-        cr = _get_cr()
-        if cr:
+
+    cr = _get_cr()
```

Comment:
walking up the stack trace is a quite expensive operation, that's why I tested first for the existence of the key (we don't expect this to happen often)

Reply:
Fair enough. I was just keeping things DRY, but this is a good reason to reverse the order and have the ``_get_cr`` call appear twice.